operations/app/src/modules/storage/main.tf

```diff
@@ -23,6 +23,120 @@ resource "azurerm_storage_account" "storage_account" {
   }
 }
 
+module "storageaccount_access_log_event_hub_log" {
+  source = "../event_hub_log"
+  resource_type = "storage_account"
+  log_type = "access"
+  eventhub_namespace_name = var.eventhub_namespace_name
+  resource_group = var.resource_group
+  resource_prefix = var.resource_prefix
+}
+
+resource "azurerm_monitor_diagnostic_setting" "storageaccount_access_log" {
+  name = "${var.resource_prefix}-storage_account-access-log"
+  # Workaround for target resource id here: https://github.com/terraform-providers/terraform-provider-azurerm/issues/8275#issuecomment-755222989
+  target_resource_id = "${azurerm_storage_account.storage_account.id}/blobServices/default/"
+  eventhub_name = module.storageaccount_access_log_event_hub_log.event_hub_name
+  eventhub_authorization_rule_id = var.eventhub_manage_auth_rule_id
+
+  log {
+    category = "StorageRead"
+    enabled  = true
+
+    retention_policy {
+      days = 0
+      enabled = false
+    }
+  }
+
+  log {
+    category = "StorageWrite"
+    enabled  = true
+
+    retention_policy {
+      days = 0
+      enabled = false
+    }
+  }
+
+  log {
+    category = "StorageDelete"
+    enabled  = true
+
+    retention_policy {
+      days = 0
+      enabled = false
+    }
+  }
+
+  metric {
+    category = "Transaction"
+    enabled = false
+
+    retention_policy {
+      days = 0
+      enabled = false
+    }
+  }
+}
+
+module "storageaccount_access_log_event_hub_log" {
+  source = "../event_hub_log"
+  resource_type = "storage_account"
+  log_type = "access"
+  eventhub_namespace_name = var.eventhub_namespace_name
+  resource_group = var.resource_group
+  resource_prefix = var.resource_prefix
+}
+
+resource "azurerm_monitor_diagnostic_setting" "storageaccount_access_log" {
```

Comment:
Is this resource supposed to be in here twice?  It looks identical to the first one.

Reply:
I think the duplicate was introduced in the merge conflicts I fixed this morning. I removed the duplicated lines.